nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -514,6 +514,30 @@ in {
               `http://hostname.domain/bucket` instead.
             '';
           };
+          useSseC = mkOption {
```

Comment:
This option should not exist, rather use whether sseCKeyFile is unset or set

Reply:
I agree

https://github.com/NixOS/nixpkgs/blob/8008e73c3878ec4b4e9633c89c5113c1cfc6abbe/nixos/modules/services/web-apps/nextcloud.nix#L517